bucket/dosbox-x.json

```diff
@@ -0,0 +1,35 @@
+{
+    "homepage": "http://dosbox-x.com",
+    "version": "0.82.19",
+    "license": "GPL-2.0-or-later",
+    "description": "DOSBox-X fork of the DOSBox project",
```

Comment:
Suggest: Fork of DOSBox, an Intel x86-based PC emulator, complete with sound, graphics, mouse, joystick, modem, etc.